ui/tests/acceptance/regions-test.js

```diff
@@ -194,29 +194,34 @@ module('Acceptance | regions (many)', function (hooks) {
     await JobsList.jobs.objectAt(0).clickRow();
     await Layout.gutter.visitClients();
     await Layout.gutter.visitServers();
-    const [
-      ,
-      ,
-      ,
-      // License request
-      // Token/policies request
-      // Search feature detection
-      regionsRequest,
-      defaultRegionRequest,
-      ...appRequests
-    ] = server.pretender.handledRequests;
```

Comment:
Previously, this was using `[, , , , ]` destructuring to say "We don't care about the first 3 requests for this test", which, while fast, is a bit haphazard, and broke as soon as we added a new request before the one being tested here. The update handles this with `.find` and `.filter`